project_app/components/ButtonGroup.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import styled from "styled-components";
+
+import Button from "@mui/material/Button";
+
+function ButtonGroup() {
```

Comment:
the naming of this file is too generic. This should be NavBarButtons or something

Reply:
Done